gnark-prover/prover/poseidon/constants.go

```diff
@@ -1,914 +1,916 @@
+	}, {
+		hex("0x193edd8e9fcf3d7625fa7d24b598a1d89f3362eaf4d582efecad76f879e36860"),
+		hex("0x18168afd34f2d915d0368ce80b7b3347d1c7a561ce611425f2664d7aa51f0b5d"),
+		hex("0x29383c01ebd3b6ab0c017656ebe658b6a328ec77bc33626e29e2e95b33ea6111"),
+	}, {
+		hex("0x10646d2f2603de39a1f4ae5e7771a64a702db6e86fb76ab600bf573f9010c711"),
+		hex("0x0beb5e07d1b27145f575f1395a55bf132f90c25b40da7b3864d0242dcb1117fb"),
+		hex("0x16d685252078c133dc0d3ecad62b5c8830f95bb2e54b59abdffbf018d96fa336"),
+	}, {
+		hex("0x0a6abd1d833938f33c74154e0404b4b40a555bbbec21ddfafd672dd62047f01a"),
+		hex("0x1a679f5d36eb7b5c8ea12a4c2dedc8feb12dffeec450317270a6f19b34cf1860"),
+		hex("0x0980fb233bd456c23974d50e0ebfde4726a423eada4e8f6ffbc7592e3f1b93d6"),
+	}, {
+		hex("0x161b42232e61b84cbf1810af93a38fc0cece3d5628c9282003ebacb5c312c72b"),
+		hex("0x0ada10a90c7f0520950f7d47a60d5e6a493f09787f1564e5d09203db47de1a0b"),
+		hex("0x1a730d372310ba82320345a29ac4238ed3f07a8a2b4e121bb50ddb9af407f451"),
+	}, {
+		hex("0x2c8120f268ef054f817064c369dda7ea908377feaba5c4dffbda10ef58e8c556"),
+		hex("0x1c7c8824f758753fa57c00789c684217b930e95313bcb73e6e7b8649a4968f70"),
+		hex("0x2cd9ed31f5f8691c8e39e4077a74faa0f400ad8b491eb3f7b47b27fa3fd1cf77"),
+	}, {
+		hex("0x23ff4f9d46813457cf60d92f57618399a5e022ac321ca550854ae23918a22eea"),
+		hex("0x09945a5d147a4f66ceece6405dddd9d0af5a2c5103529407dff1ea58f180426d"),
+		hex("0x188d9c528025d4c2b67660c6b771b90f7c7da6eaa29d3f268a6dd223ec6fc630"),
+	}, {
+		hex("0x3050e37996596b7f81f68311431d8734dba7d926d3633595e0c0d8ddf4f0f47f"),
+		hex("0x15af1169396830a91600ca8102c35c426ceae5461e3f95d89d829518d30afd78"),
+		hex("0x1da6d09885432ea9a06d9f37f873d985dae933e351466b2904284da3320d8acc"),
+	}, {
+		hex("0x2796ea90d269af29f5f8acf33921124e4e4fad3dbe658945e546ee411ddaa9cb"),
+		hex("0x202d7dd1da0f6b4b0325c8b3307742f01e15612ec8e9304a7cb0319e01d32d60"),
+		hex("0x096d6790d05bb759156a952ba263d672a2d7f9c788f4c831a29dace4c0f8be5f"),
+	}, {
+		hex("0x054efa1f65b0fce283808965275d877b438da23ce5b13e1963798cb1447d25a4"),
+		hex("0x1b162f83d917e93edb3308c29802deb9d8aa690113b2e14864ccf6e18e4165f1"),
+		hex("0x21e5241e12564dd6fd9f1cdd2a0de39eedfefc1466cc568ec5ceb745a0506edc"),
+	}, {
+		hex("0x1cfb5662e8cf5ac9226a80ee17b36abecb73ab5f87e161927b4349e10e4bdf08"),
+		hex("0x0f21177e302a771bbae6d8d1ecb373b62c99af346220ac0129c53f666eb24100"),
+		hex("0x1671522374606992affb0dd7f71b12bec4236aede6290546bcef7e1f515c2320"),
+	}, {
+		hex("0x0fa3ec5b9488259c2eb4cf24501bfad9be2ec9e42c5cc8ccd419d2a692cad870"),
+		hex("0x193c0e04e0bd298357cb266c1506080ed36edce85c648cc085e8c57b1ab54bba"),
+		hex("0x102adf8ef74735a27e9128306dcbc3c99f6f7291cd406578ce14ea2adaba68f8"),
+	}, {
+		hex("0x0fe0af7858e49859e2a54d6f1ad945b1316aa24bfbdd23ae40a6d0cb70c3eab1"),
+		hex("0x216f6717bbc7dedb08536a2220843f4e2da5f1daa9ebdefde8a5ea7344798d22"),
+		hex("0x1da55cc900f0d21f4a3e694391918a1b3c23b2ac773c6b3ef88e2e4228325161"),
+	},
+}
+
+var MDS_4 = [][]frontend.Variable{
```

Comment:
please remove the poseidon constants which are not working.
Imo. we should solve that in a different pr and until then not have this code in main since it's going to be confusing.